File: PosBiptek/app/src/main/java/com/biptek/posbiptek/model/ListProdukRestock.java
package com.biptek.posbiptek.model;

public class ListProdukRestock {
    private long id_list_produk_restock;
    private long kode_restock_list;
    private String kode_produk_list_restock;
    private int jumlah_produk_restock;

    public ListProdukRestock(long id_list_produk_restock, long kode_restock_list, String kode_produk_list_restock, int jumlah_produk_restock) {
        this.id_list_produk_restock = id_list_produk_restock;
        this.kode_restock_list = kode_restock_list;
        this.kode_produk_list_restock = kode_produk_list_restock;
        this.jumlah_produk_restock = jumlah_produk_restock;
    }

    public ListProdukRestock(String kode_produk_list_restock, int jumlah_produk_restock) {
        this.kode_produk_list_restock = kode_produk_list_restock;
        this.jumlah_produk_restock = jumlah_produk_restock;
    }

    public ListProdukRestock() {

    }

    public long getId_list_produk_restock() {
        return id_list_produk_restock;
    }

    public void setId_list_produk_restock(long id_list_produk_restock) {
        this.id_list_produk_restock = id_list_produk_restock;
    }

    public long getKode_restock_list() {
        return kode_restock_list;
    }

    public void setKode_restock_list(long kode_restock_list) {
        this.kode_restock_list = kode_restock_list;
    }

    public String getKode_produk_list_restock() {
        return kode_produk_list_restock;
    }

    public void setKode_produk_list_restock(String kode_produk_list_restock) {
        this.kode_produk_list_restock = kode_produk_list_restock;
    }

    public int getJumlah_produk_restock() {
        return jumlah_produk_restock;
    }

    public void setJumlah_produk_restock(int jumlah_produk_restock) {
        this.jumlah_produk_restock = jumlah_produk_restock;
    }

    @Override
    public String toString() {
        return "ListProdukRestock{" +
                "kode_restock_list=" + kode_restock_list +
                ", kode_produk_list_restock='" + kode_produk_list_restock + '\'' +
                ", jumlah_produk_restock=" + jumlah_produk_restock +
                '}';
    }
}

File: Server/advanced/backend/models/ListProdukRestock.php
<?php

namespace backend\models;

use Yii;

/**
 * This is the model class for table "list_produk_restock".
 *
 * @property int $id_list_produk_restock
 * @property int $kode_restock_list
 * @property string $kode_produk_list_restock
 * @property int $jumlah_produk_restock
 *
 * @property Restock $kodeRestockList
 * @property Produk $kodeProdukListRestock
 */
class ListProdukRestock extends \yii\db\ActiveRecord
{
    /**
     * {@inheritdoc}
     */
    public static function tableName()
    {
        return 'list_produk_restock';
    }

    /**
     * {@inheritdoc}
     */
    public function rules()
    {
        return [
            [['kode_restock_list', 'kode_produk_list_restock', 'jumlah_produk_restock'], 'required'],
            [['kode_restock_list', 'jumlah_produk_restock'], 'integer'],
            [['kode_produk_list_restock'], 'string', 'max' => 13],
            [['kode_restock_list'], 'exist', 'skipOnError' => true, 'targetClass' => Restock::className(), 'targetAttribute' => ['kode_restock_list' => 'kode_restock']],
            [['kode_produk_list_restock'], 'exist', 'skipOnError' => true, 'targetClass' => Produk::className(), 'targetAttribute' => ['kode_produk_list_restock' => 'kode_produk']],
        ];
    }

    /**
     * {@inheritdoc}
     */
    public function attributeLabels()
    {
        return [
            'id_list_produk_restock' => 'Id List Produk Restock',
            'kode_restock_list' => 'Kode Restock List',
            'kode_produk_list_restock' => 'Kode Produk List Restock',
            'jumlah_produk_restock' => 'Jumlah Produk Restock',
        ];
    }

    /**
     * @return \yii\db\ActiveQuery
     */
    public function getKodeRestockList()
    {
        return $this->hasOne(Restock::className(), ['kode_restock' => 'kode_restock_list']);
    }

    /**
     * @return \yii\db\ActiveQuery
     */
    public function getKodeProdukListRestock()
    {
        return $this->hasOne(Produk::className(), ['kode_produk' => 'kode_produk_list_restock']);
    }
}
